zoo-css-challenge/style.css

```diff
@@ -106,8 +120,16 @@ footer {
 .image-container img {
   vertical-align: top;
   border: 5px solid #747c92;
+  height: 250px;
+  width: 270px;
+
 }
 
+.image-container{
+display: flex;
+flex-direction: row;
+justify-content: space-between;
+}
 .image-container img:first-of-type {
```

Comment:
Try to keep your css clean by identifying style rules and declarations that no longer have an effect and remove them like this one here.